psalm.xml.dist

```diff
@@ -14,4 +14,9 @@
             <directory name="vendor" />
         </ignoreFiles>
     </projectFiles>
+    <stubs>
```

Comment:
These stub files are temporary until we add stubs for new classes to psalm itself.